neurodsp/tests/test_spectral.py

```diff
@@ -0,0 +1,101 @@
+"""
+test_timefreq.py
+Test functions in the time-frequency analysis module
+"""
+
+import pytest
```

Comment:
You don't need to import pytest for a test file.

Reply:
True. This should be changed for all test files.